src/session_pool/session.js

```diff
@@ -86,6 +91,7 @@ export class Session {
         this.errorScore = errorScore; // indicates number of markBaded request with the session
         this.maxUsageCount = maxUsageCount;
         this.sessionPool = sessionPool;
+        this.type = type;
```

Comment:
It feels to me quite risky and error-prone to allow storing of cookies in either format, and have this extra `type` attribute. It could be a source of esoteric bugs. Why not just store cookies in the tough-cookie format always? And then just have functions `getPuppeteerCookies` and `getToughCookies` that will return data in correct format?

Reply:
I would prefer the two `get` functions too. I just was not sure about the consistency with one set function and two get functions.

Reply:
Fair enough, so maybe we can really have `setPuppeteerCookies` and `setToughCookies`? It will be consistent

Reply:
sounds good to me 👍

Reply:
Cool, whatever you think makes most sense.